jekyll/_cci2/using-macos.md

```diff
@@ -92,55 +93,37 @@ Running or testing Apple Silicon apps natively is currently not possible as Circ
 ### Universal Binaries
 {: #universal-binaries }
 
-Xcode currently supports the creation of universal
-binaries which can be run on both `x86_64` and `ARM64` CPU architectures without
-needing to ship separate executables. This is supported only under Xcode 12.2+,
-although older Xcode versions can still be used to compile separate `x86_64` and
-`ARM64` executables.
+Xcode currently supports the creation of universal binaries which can be run on both `x86_64` and `ARM64` CPU architectures without needing to ship separate executables. This is supported only under Xcode `12.2`+, although older Xcode versions can still be used to compile separate `x86_64` and `ARM64` executables.
 
-### Extracting Unwanted Architectures
-{: #extracting-unwanted-architectures }
+### Extract unwanted architectures
+{: #extract-unwanted-architectures }
 
-By default, Xcode 12.2+ will create universal binaries, compiling to a single
-executable that supports both `x86_64` and `ARM64` based CPUs. If you need to remove
-an instruction set, you can do so by using the `lipo` utility.
+By default, Xcode `12.2`+ will create universal binaries, compiling to a single executable that supports both `x86_64` and `ARM64` based CPUs. If you need to remove an instruction set, you can do so by using the `lipo` utility.
 
-Assuming that you want to create a standalone `x86_64` binary from a
-universal binary called `circleci-demo-macos`, you can do so by running the
-command:
+Assuming that you want to create a standalone `x86_64` binary from a universal binary called `circleci-demo-macos`, you can do so by running the command:
 
 ```shell
 lipo -extract x86_64 circleci-demo-macos.app/Contents/MacOS/circleci-demo-macos -output circleci-demo-macos-x86_64
 ```
 
-You can then confirm the supported architecture of the extracted binary with
-`lipo -info circleci-demo-macos-x86_64`, which will output the following
+You can then confirm the supported architecture of the extracted binary with `lipo -info circleci-demo-macos-x86_64`, which will output the following:
```

Comment:
`lipo -info circleci-demo-macos-x86_64` should be in it's own code block, and I think using `shell` formatting is correct for all three items here - two runnable commands and one example output in the terminal.